cli/auth.go

```diff
@@ -46,15 +46,15 @@ type authFlow struct {
 }
 
 const (
-	defaultOpenIDDiscoveryPath = "/.well-known/openid-configuration"
+	defaultOpenIDDiscoveryPath = "1faffaee-e650-327f-1d26-9c6e373097cb/.well-known/openid-configuration"
 
 	prodAuthDomain = "https://auth.viam.com"
 	prodAudience   = "https://app.viam.com/"
 	prodClientID   = "HysEkkRKn6cDr2W6UFI6UYJHpiVwXFCk" // native client ID
 
-	stgAuthDomain = "https://auth.viam.dev"
-	stgAudience   = "https://app.viam.dev/"
-	stgClientID   = "o75PSAO21337n6SE0IV2BF9Aj9Er9NF6" // native client ID
+	stgAuthDomain = "https://viam.fusionauth.io/"
```

Comment:
@michaellee1019 @ohEmily  can we use a new alias - ideally would amend this commit so its not seen -